rds.tf

```diff
@@ -1,7 +1,8 @@
 locals {
-  rds_identifier = length(var.rds_identifier_override) > 0 ? var.rds_identifier_override : var.name
-  rds_db_name    = length(regexall("sqlserver-.*", var.rds_engine)) > 0 ? null : (length(var.rds_dbname_override) > 0 ? var.rds_dbname_override : local.rds_identifier)
-  password       = var.rds_use_random_password ? join("", random_string.password.*.result) : var.rds_admin_pass
+  rds_identifier                = length(var.rds_identifier_override) > 0 ? var.rds_identifier_override : var.name
+  rds_db_name                   = length(regexall("sqlserver-.*", var.rds_engine)) > 0 ? null : (length(var.rds_dbname_override) > 0 ? var.rds_dbname_override : local.rds_identifier)
+  password                      = var.rds_use_random_password ? join("", random_string.password.*.result) : var.rds_admin_pass
+  rds_final_snapshot_identifier = length(var.rds_final_snapshot_identifier_override) > 0 ? "${var.rds_final_snapshot_identifier_override}-snapshot" : "${var.env}-${local.rds_identifier}-snapshot"
```

Comment:
I think in this case if `rds_final_snapshot_identifier_override` is used - there should be no `-snapshot` suffix.